src/selective.d

```diff
@@ -202,15 +204,33 @@ private bool isPathExcluded(string path, string[] allowedPaths)
 	path = buildNormalizedPath(path);
 	foreach (allowed; allowedPaths) {
 		auto comm = commonPrefix(path, allowed);
+		// What are we comparing against?
+		log.vdebug("Evaluation against 'sync_list' entry: ", allowed);
+		// is path is an exact match
 		if (comm.length == path.length) {
 			// the given path is contained in an allowed path
+			log.vdebug("Evaluation against 'sync_list' result - direct match");
 			return false;
 		}
+		// is path is a subitem
 		if (comm.length == allowed.length && path[comm.length] == '/') {
 			// the given path is a subitem of an allowed path
+			log.vdebug("Evaluation against 'sync_list' result - parental path match");
 			return false;
 		}
+		// is the allowed path terminated by a wildcard? (*)
+		if (allowed.endsWith("*")) {
```

Comment:
That gives at least a partial answer for that, but `*` isn't allowed in any other location, right? OTOH, we have 
```
Config option 'skip_file'              = ~*|.~*|*.tmp|*.swp
```
so `*` are evaluated there also in other positions.

I fear that people will use synclist entries like
```
random_files/img-201*-*-*/raw/
```
(just a dummy example) and expect it to work.

I am not sure what a good solution is. I see a few options, either something like the `.gitignore` configuration style, or some `find` syntax style.

But well, at least this is a good step, albeit we need to make clear that the star only works at the end of the entry, I guess.

Reply:
Yes.. the current documentation only shows '*' at end, could updated it to make it super clear that this is the only option, or play around with a search if '*' is present, then do a regex to search & match ... would be computationally expensive depending on how many '*' are being pattern matched.

Will look at it for a while and see what I can come up with.

Reply:
Should be resolved by https://github.com/abraunegg/onedrive/pull/1063/commits/191571e5c15f916aeb7ddaf3c148d589957eb844